git_stalk/stalk.py

```diff
@@ -20,7 +20,7 @@ def jft(user):
 
 
 def get_event(string):
-    """Returns the event"""
+    # Returns the event
```

Comment:
Revert this everywhere in the file. This is called docstring. Please read:  [https://www.python.org/dev/peps/pep-0257/#id16](One line docstring)